etc/eagle_config.json

```diff
@@ -32,8 +23,24 @@
             }
         ]
     },
-
+    "capacity": {
+        "enabled":  true,
```

Comment:
I thought about situation when we have need to temporary disable existing and configured service - so at least endpoint is remaining in config file for future needs.

I will remove all "enabled" params